### ISSUE TITLE
Clarity on available templates and variables

### ISSUE THREAD

**Issue by CryptoMaximalist** [FoxxMD/context-mod#121]
One other thing I've been unsure about has been the available options to use in a {{mustache.template}}.  The first two are intuitive, {{rule.rulename}}, however I never know what the final dotwalk can be unless it's been in an example.

For example, I haven't seen any docs about variables available for ModNote rules and the variables here resolves as blank. 

```
          - kind: ban
            authorIs:
              include:
                - name: hasEnoughIPModNotes
                  criteria:
                    modActions:
                      - note: '/insufficient participation/i'
                        noteType:
                          - SPAM_WARNING
                        search: total
                        count: "> 2 in 1 year"
            duration: 14
            message: >-
              **User has had {{rules.hasenoughipmodnotes.totalCount}} warnings

              Summary- {{rules.hasenoughipmodnotes.thresholdSummary}}
```

Is this info somewhere in the schema that I've been missing?  It'd be cool if this info was in more of the docs and examples. Maybe even it could be an autocomplete function in the editor

**Comment by FoxxMD** [comment]
There are some common template variables that are always available regardless of what rules/actions you've run. [They can be found here](https://github.com/FoxxMD/context-mod/blob/master/docs/subreddit/actionTemplating.md) alongside examples of how to use the templating in general.

As for specific rules...some of them can be found in the [schema documentation](https://json-schema.app/view/%23/%23%2Fdefinitions%2FRunConfigData/%23%2Fdefinitions%2FSubmissionCheckConfigData?url=https%3A%2F%2Fraw.githubusercontent.com%2FFoxxMD%2Freddit-context-bot%2Fmaster%2Fsrc%2FSchema%2FApp.json) for specific rules but overall they are basically undocumented :sweat_smile: They were changing so much in early development I didn't put in the effort to document at the time and now they've been neglected. I will make sure to get docs in for all of them soon!

**Comment by FoxxMD** [comment]
@CryptoMaximalist On the edge branch you can now find template variable examples for most rules in the README files in each rule folder from [here](https://github.com/FoxxMD/context-mod/tree/edge/docs/subreddit/components) -- [example](https://github.com/FoxxMD/context-mod/tree/edge/docs/subreddit/components/recentActivity#template-variables)

**Comment by FoxxMD** [comment]
Examples now included in release [0.13.3](https://github.com/FoxxMD/context-mod/releases/tag/0.13.3)
